### DISCUSSION TITLE
External database

### DISCUSSION THREAD

**Discussion by eZwzAEkx** [WPCloudDeploy/wp-cloud-deploy#14] [General]
Hi,

do you support an external database when creating WordPress installations?

**Comment by elindydotcom** [comment]
Hi:
Thanks for your question but this is not a support or sales forum.  It's a development forum and these kinds of messages aren't allowed in here since they're not related to development of the product.  

There are four places you can submit your question to and get an answer in a reasonable amount of time:

- [The private FB group](https://www.facebook.com/groups/wp.linux.support)  
- The sales folks at sales (at) wpclouddeploy dot com
- A [twitter message](https://twitter.com/wpclouddeploy), either public or private
- A FB private message via the [public FB page](https://www.facebook.com/wpclouddeploy).

There are probably a couple of avenues I'm forgetting but choosing one of those should get a response.

Thanks.
